app/services/statements/query.rb

```diff
@@ -1,19 +1,39 @@
 module Statements
   class Query
-    def initialize(lead_provider:)
-      @lead_provider = lead_provider
+    def initialize
+      @scope = Statement.all
+    end
+
+    def by_lead_provider(lead_provider)
```

Comment:
`#by_lead_provider` sounds like they'll be grouped by lead provider.

Maybe `#belonging_to_lead_provider`, or even `#belonging_to(lead_provider:)`?

Reply:
preference is to have lead provider in the method name, as to avoid someone later adding more to this method to keep it explicit